framework/db/pgsql/QueryBuilder.php

```diff
@@ -64,6 +70,27 @@ class QueryBuilder extends \yii\db\QueryBuilder
         'NOT EXISTS' => 'buildExistsCondition',
     ];
 
+    /**
+     * Builds a SQL statement for creating a new index.
+     * @param string $name the name of the index. The name will be properly quoted by the method.
+     * @param string $table the table that the new index will be created for. The table name will be properly quoted by the method.
+     * @param string|array $columns the column(s) that should be included in the index. If there are multiple columns,
+     * separate them with commas or use an array to represent them. Each column name will be properly quoted
+     * by the method, unless a parenthesis is found in the name.
+     * @param boolean|string $index whether to add UNIQUE constraint or create another index.
```

Comment:
Oh, now I got it, this can be a string now to indicate the index type. Better rename it to indexType and add explaination how this parameter can be used.